pypuck/pypuck.py

```diff
@@ -139,7 +139,26 @@ def team_stats(start_season=None, end_season=None):
     Edmonton Oilers |  34  |   23   |  5 |   73   | ...
     ------------------------------------------------
     """
-    pass
+    
+    base_url = 'https://api.nhle.com/stats/rest/en/team/summary?'
+    arguments = 'cayenneExp=gameTypeId=2' +\
+          f' and seasonId<={end_season}' +\
+          f' and seasonId>={start_season}'
+    if str.isdecimal(start_season) == False:
+      print("Invalid Input. Enter start_season in Bi-Annual format. 'YYYYYYYY'")
+      return
+    if str.isdecimal(end_season) == False:
+      print("Invalid Input. Enter end_season in Bi-Annual format. 'YYYYYYYY'")
+      return
```

Comment:
I made a helper function that can handle this error, so you can add this to the code instead.

```suggestion
    helpers.check_argument_type(start_season, 'start_season', str)
    helpers.check_argument_type(end_season, 'end_season', str)
```